Lib/idlelib/tree.py

```diff
@@ -410,42 +422,68 @@ def GetSubList(self):
 # A canvas widget with scroll bars and some useful bindings
 
 class ScrolledCanvas:
+
```

Comment:
Reformatting existing code is not part of this issue and a distraction from the real implementation problems.  I would do it differently, but on another issue sometime in the future.